OpenQuestion/answer-question-2.md

```diff
@@ -0,0 +1,12 @@
+In HTML and CSS, different tags have different default display properties:
+1. **Block-Level Elements**:
+   - Block-level elements always start on a new line and take up the full width available.
+   - Examples of block-level elements include `<div>`, `<p>`, `<h1>` to `<h6>`, `<ul>`, `<ol>`, `<li>`, `<table>`, and more.
+2. **Inline Elements**:
+   - Inline elements do not start on a new line and only take up as much width as necessary.
+   - Examples of inline elements include `<span>`, `<a>`, `<strong>`, `<em>`, `<img>`, `<input>`, and more.
+3. **Inline-Block Elements**:
+   - Inline-block elements behave like inline elements but allow you to set a width and height.
+   - They also respect top and bottom margins/paddings.
+   - Examples include `<button>`, `<label>`, and sometimes `<img>`.
+   ![alt text](image.png)
```

Comment:
`alt text` is not good, please recheck what is alt attribute

Reply:
I fixed it